docs/wiki/research/PBS/mev-boost.md

```diff
@@ -0,0 +1,56 @@
+<!-- @format -->
+
+# Mev-boost: A Popular PBS Implementation
+
+[comment]: <> (This is still unrefined)
+
+Mev-boost is a widely recognized open-source implementation of Proposer-Builder Separation (PBS) for Ethereum. It allows validators to outsource block building to specialized builders, potentially leading to increased validator rewards and improved network efficiency.
+Here's how mev-boost works:
+
+<figure style="text-align: center;">
+  <img src="images/pbs/block-building.png" alt="PBS">
+  <figcaption style="text-align: center;">Current Block building. Source: <a href="https://barnabe.substack.com/p/pbs">Baranabé's article</a></figcaption>
+</figure>
+
+1. **Block Building:**
+   Specialized builders, known as "searchers" in mev-boost, compete to create the most profitable block for the proposer. They do this by optimizing transaction ordering and inclusion, taking into account factors like gas fees, transaction priority, and potential MEV (Maximal Extractable Value).
```

Comment:
Follow consistent linking throughout the article. 

- In the MEV file, when mentioning PBS, link it to the PBS file. 
- In the PBS file, when discussing MEV-Boost, link it to the MEVBoost file.